Makefile

```diff
@@ -35,8 +35,12 @@ CONFIG_VARS = curl -fsL https://ft-next-config-vars.herokuapp.com/$1/$(call APP_
 
 # clean
 clea%:
-# HACK: Can't use -e option here because it's not supported by our Jenkins
-	@git clean -fxd
+	# clean just untracked files if not on CircleCI
+	@if [ -z $(CIRCLECI) ]; then git clean -Xfd; else git clean -xfd; fi
```

Comment:
Yep :) so my only comment now is that the conditional is confusing (at least for me). Can it instead read:

```
@if [ -e $(CIRCLECI) ]; then git clean -xfd; else git clean -Xfd; fi
```